src/main/java/net/neoforged/neoforge/client/gui/ModMismatchDisconnectedScreen.java

```diff
@@ -137,25 +181,31 @@ private List<Pair<FormattedCharSequence, FormattedCharSequence>> splitLineToWidt
         }
 
         /**
-         * Adds a style information to the given mod name string. The style assigned to the returned component contains the color of the mod name,
-         * a hover event containing the given id, and an optional click event, which opens the homepage of mod, if present.
+         * Uses the given channel id to return a component with the name of the mod that likely owns the channel. If no such mod is found, the namespace of the channel id is used instead.
```

Comment:
See my comment above, I don't really support this "merge and blame" approach in this interface. Modders are free and explicitly requested the ability to create channels in other namespaces that are not their own.